### PR TITLE
XIVY-5656 convert demos to latest project version

### PR DESCRIPTION
**Pull request by ivy-lli**

### CODE REVIEW

File: error-handling-demos/config/databases.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Databases:
+  stock:
+    Url: jdbc:hsqldb:mem:StockDb
+    Driver: org.hsqldb.jdbcDriver
+  invalidDB:
+    Url: jdbc:postgresql://null/InvalidDB
+    Driver: org.postgresql.Driver
+    UserName: ''
+    Properties:
+      This DB is invalid on purpose: so it can be used to throw an error
```
Comment (alexsuter) [inline error-handling-demos/config/databases.yaml:10]: hm.. does this work?

**Review by alexsuter — APPROVED** [review]

File: error-handling-demos/config/databases.yaml (hunk 1 above)
Comment (ivy-lli) [reply error-handling-demos/config/databases.yaml:10]: haha good eye 🙈
I will check that 😉

Comment (ivy-lli) [reply error-handling-demos/config/databases.yaml:10]: grafik
grafik
grafik

Comment (ivy-lli) [reply error-handling-demos/config/databases.yaml:10]: Looks good, I think

Comment (ivy-lli) [reply error-handling-demos/config/databases.yaml:10]: Same what happens on 9.2 engine 👍

**Pull request merged**